python/lambda-from-container/README.md

```diff
@@ -0,0 +1,79 @@
+Lambda Function from Container
+=============================
+
+* This example will produce a **Lambda function** from a **Container Image**.
```

Comment:
An explanation of why this is a useful patter (or a description of use-case) would really help here